ix-dev/community/lyrion-music-server/questions.yaml

```diff
@@ -0,0 +1,560 @@
+groups:
+  - name: LMS Configuration
+    description: Configure Lyrion Music Server
+  - name: Network Configuration
+    description: Configure Network for Lyrion Music Server
+  - name: Storage Configuration
+    description: Configure Storage for Lyrion Music Server
+  - name: Labels Configuration
+    description: Configure Labels for Lyrion Music Server
+  - name: Resources Configuration
+    description: Configure Resources for Lyrion Music Server
+
+questions:
+  - variable: TZ
+    group: LMS Configuration
+    label: Timezone
+    schema:
+      type: string
+      default: Etc/UTC
+      required: true
+      $ref:
+        - definitions/timezone
+
```

Comment:
missing the `variable: lms` section with the addtitional envs section

Reply:
done.